stores/menu.ts

```diff
@@ -191,6 +191,9 @@ export const menuStore = defineStore('menu', {
           )
         ).filter(e => e) as ApiPois[]
 
+        const { flattenMultipoint } = useGeometry()
+        posts = posts.map(p => ({ ...p, features: flattenMultipoint(p.features) as ApiPoi[] }))
```

Comment:
No need for composable here, as std import is fine. Or even move the code here as it the only use.